packages/@lwc/integration-karma/test/synthetic-shadow/scoped-id/href.spec.js

```diff
@@ -60,11 +62,37 @@ it('should not transform fragment-only urls when the template has no ids', () =>
     const elm = createElement('x-href-dangling', { is: HrefDangling });
     document.body.appendChild(elm);
     expect(elm.shadowRoot.querySelector('a').getAttribute('href')).toBe('#foo');
+    expect(elm.shadowRoot.querySelector('area').getAttribute('href')).toBe('#bar');
 });
 // Enable this test when we transform all href values with fragment-only urls
-// https://github.com/salesforce/lwc/issues/1150
+// TODO [#1150]: Always mangle frag-id href attribute values
```

Comment:
Completely unrelated to this PR--I originally thought that this inconsistent behavior would cause problems but in practice, it hasn't (at least we've never been made aware of any issues).